superset-frontend/cypress-base/cypress/integration/explore/visualizations/bubble.test.js

```diff
@@ -57,7 +57,10 @@ describe('Visualization > Bubble', () => {
     cy.route('POST', '/superset/explore_json/**').as('getJson');
   });
 
-  it('should work', () => {
+  // Number of circles are pretty unstable when there are a lot of circles
+  // Since main functionality is already covered in fitler test below,
+  // skip this test untill we find a solution.
```

Comment:
if this test is unstable and we already cover main functionality, should we delete the test? Seems like there's not much point in keeping it around tbh

Reply:
Part of me still wants to find the root cause of this instability, even though I may never find time to do it, but it still doesn't hurt to keep this here as a reminder?